src/components/Details/Details.css

```diff
@@ -0,0 +1,44 @@
+:root {
+  --dark-blue: #1a3871;
+  --white: rgb(255, 255, 255);
+  --lightblue: #4872c2;
+}
+
+body {
+  background-color: var(--lightblue);
+  color: var(--white);
+}
+
+.detail-data {
+  display: flex;
+  flex-direction: column;
+  width: 80%;
+  margin: auto;
+  row-gap: 20px;
+}
+
+.covid-data {
+  display: flex;
+  justify-content: space-between;
+  background-color: var(--dark-blue);
+  height: 35px;
+  align-items: center;
+}
+
+.flag {
```

Comment:
- [ ] Kindly, add a margin to the text to match the design.

![rev1](https://user-images.githubusercontent.com/53347966/204102338-fb55475d-ad5e-477a-8d58-c04b1c29b1dd.JPG)